dist/index.js

```diff
@@ -0,0 +1,98 @@
+//didnt understood why i should make it in a seperated Class if i can use it as a regular function **(תאריך תפוגה (מחלקה נפרדת))**
+function randomDate(start, end) {
+    return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+}
+class Product {
+    constructor(name, price, weight, isKosher, manufacturer) {
+        this.name = name;
+        this.price = price;
+        this.weight = weight;
+        this.isKosher = isKosher;
+        this.manufacturer = manufacturer;
+        this.date = randomDate(new Date(2012, 0, 1), new Date(2025, 0, 1));
+    }
+    // console logged and inserted to the Table
+    get showProductFull() {
+        return console.log(`Name:${this.name} price:$${this.price} weight:${this.weight}Kg  isKoser? ${this.isKosher}
+        manufacturer :${this.manufacturer}`);
+    }
+    // console logged and inserted to the Table 
+    get showProductNameAndPrice() {
+        return console.log(`Name:${this.name} price:$${this.price} `);
+    }
+    //  priceforKillo
+    priceForKilo() {
+        return (this.price / this.weight);
+    }
+    //higher price
+    returnHigherPrice(pro1, pro2) {
```

Comment:
this method should not be inside the class. only relevant in case of static method